api/dataconfigurator/dataconfigurator.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package dataconfigurator;
+
+option go_package = "github.com/ligato/vpp-agent/api/dataconfigurator;dataconfigurator";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+option (gogoproto.messagename_all) = true;
+
+import "models/vpp/vpp.proto";
+import "models/linux/linux.proto";
+
+message ConfigData {
+    vpp.Config vpp = 1;
+    linux.Config linux = 2;
+}
+
+service DataConfigurator {
+    rpc Resync(ConfigData) returns (ResyncResponse);
```

Comment:
Don't take this comment to seriously... but if you:

Resync -> Create + Delete (${some value for ConfigData that deletes *all* data)
Get -> Read

You are left with the super familiar 'CRUD' ...

Reply:
Yeah. Plan is to hide _resync_ from outside, since it's really not common way to to stuff.

Reply:
All good :)